apiservers/engine/backends/container.go

```diff
@@ -33,7 +33,8 @@ import (
 	"github.com/docker/engine-api/types/container"
 	"github.com/docker/engine-api/types/strslice"
 
-	"github.com/vmware/vic/apiservers/portlayer/client/exec"
+	"github.com/vmware/vic/apiservers/portlayer/client/containers"
```

Comment:
Trying to follow REST here. exec exposes a collection of objects from the PL, just like network. REST operates on resources/objects, so we have /containers and /scopes there instead of /exec/containers and /network/scopes.